src/libs/blockchain/gcr/gcr_routines/handleNativeOperations.ts

```
@@ -43,10 +49,127 @@ export class HandleNativeOperations {
+                        // The SDK will extract it from the tx response
+                    } catch (tokenError) {
+                        log.error(`[TLSNotary] Failed to create token: ${tokenError}`)
+                        // Continue - the fee was already burned, token creation failure is logged
+                    }
+                }
+                break
+
+            // REVIEW: TLSNotary proof storage - burns fee based on size, stores proof
+            case "tlsn_store":
+                // eslint-disable-next-line no-var
+                var [tokenId, proof, storageType] = nativePayload.args
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)